src/Tickets/Blocks/Tickets/app/editor/container/container.js

```diff
@@ -94,6 +94,11 @@ const mapStateToProps = (state, ownProps) => {
 		{ state, ownProps }
 	);
 
+	applyFilters(
```

Comment:
Why filter again? The mapped props have just been filtered, hook in the previous filtering call to filter them.

Furthermore: I'm not clear about the intent here: the `mappedProps` value is the object containing the mapped properties of the component that has been filtered above; that object will be subject of the filter application, not the prompt.